### ISSUE TITLE
Category should be a string or array of strings (not h-card)

### ISSUE THREAD

**Issue by jackjamieson2** [jackjamieson2/yarns-microsub-server#98]
This post (https://boffosocko.com/2020/01/16/how-to-follow-the-complete-output-of-journalists-and-other-writers/) contains an h-card marked up as `u-category`.  When parsed, it results in an h-card inside the category field, which isn't supported.

In alltogethernow.io, this means the entire channel's posts fail to load

In Monocle the post loads, but each of the h-card fields is displayed as a separate category. This results in a category field called `card` 

If categories are parsed as h-cards they should be normalized to strings or arrays of strings. This can probably be done in parse this.

(Truthfully, I'm behind in updating Yarns so I should check if this has already been addressed in Parse This and update)

**Comment by dshanske** [comment]
https://indieweb.org/person-tag#How_to_markup That would be a person tag.

**Comment by dshanske** [comment]
I checked and xray actually collapses it and puts the h-card into the reference section, as noted in the jf2 spec. Parse This should probably do the same, but not a scenario I considered

**Comment by jackjamieson2** [comment]
Only seems to come up rarely.  Turns out to be a large problem when viewing
feeds in Together though, since the whole channel fails if any feed item
has field of the wrong type


On January 21, 2020 at 2:07:52 PM, David Shanske (notifications@github.com)
wrote:

I checked and xray actually collapses it and puts the h-card into the
reference section, as noted in the jf2 spec. Parse This should probably do
the same, but not a scenario I considered

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/jackjamieson2/yarns-microsub-server/issues/98?email_source=notifications&email_token=AC5U5R7AOG27A2Y4EV3JLWLQ65BYRA5CNFSM4KJYMC42YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJQ4F7Y#issuecomment-576832255>,
or unsubscribe
